.flake8

```diff
@@ -227,3 +267,42 @@ per-file-ignores =
   tests/unit/image_manager/__init__.py: D104
   tests/unit/__init__.py: D104
   share/ansible_navigator/utils/__init__.py: D104
+
+# Count the number of occurrences of each error/warning code and print a report:
+statistics = true
+
+# flake8-eradicate
+# E800:
+eradicate-whitelist-extend = isort:\s+\w+
+
+# flake8-pytest-style
+# PT001:
+pytest-fixture-no-parentheses = true
+# PT006:
+pytest-parametrize-names-type = tuple
+# PT007:
```

Comment:
We currently do not have any violations of this, can we reconsider when we do or it comes up?

Reply:
This is not an ignore, it's a plugin setting.

Reply:
got it!